src/FluentMigrator.Runner.Core/Conventions/DefaultPrimaryKeyNameConvention.cs

```diff
@@ -29,23 +32,24 @@ public class DefaultPrimaryKeyNameConvention : IColumnsConvention
         /// <inheritdoc />
         public IColumnsExpression Apply(IColumnsExpression expression)
         {
+            if (expression.Columns.Count(x => x.IsPrimaryKey) > 1)
```

Comment:
I don't understand why this is something that should be discouraged. It is perfectly valid for a table to have a primary key with multiple columns.

Reply:
One primary key on many columns is good. But here we are talking specifically about multiple primary keys, each with its own name, am I wrong?

Reply:
I am pretty sure you are wrong. But your questioning my statement honestly gives me pause. But I'm 99% sure if I integration test this on a compound key, it will fail.